_data/compose-cli/docker_compose_events.yaml

```diff
@@ -19,7 +19,7 @@ long: |-
     }
     ```
 
-    The events that can be received using this can be seen [here](/engine/reference/commandline/events/#object-types).
+    The events that can be received using this can be seen [here](https://docs.docker.com/engine/reference/commandline/events/#object-types).
```

Comment:
Temporary fix (needs to be fixed in upstream, or need to figure out why it prepends `/engine/reference/commandline/` to the URL)

Reply:
Yup, makes sense